modules/apps/change-tracking/change-tracking-change-lists-configuration-web/src/main/resources/content/Language.properties

```diff
@@ -1,6 +1,7 @@
 an-error-occured-while-saving-configuration=An error occured while saving the configuration.
 change-lists-help=Change Lists are flexible way to manage different versions of sites including the assets and the display of the assets with utilizing a sandbox concept. This means the users can essentially group certain changes into one set and publish those changes whenever they are ready.
 enable-change-lists=Enable Change Lists
+javax.portlet.title.com_liferay_change_tracking_web_portlet_ChangeListsConfigurationPortlet=Settings
```

Comment:
@dantewang @ChiZelin 
ChangeListsConfigurationPortlet's package name is "com.liferay.change.tracking.change.lists.configuration.web.internal.portlet", the key name here is not consistent with package name.

Tina.

Reply:
@tinatian Leon did a check among the portlets affected by this PR, and `com_liferay_exportimport_web_portlet_ChangesetPortlet` is also inconsistent with its package `com.liferay.exportimport.changeset.web.internal.portlet`.

However, unlike change-tracking ones which are still in development, this portlet is in 7.1. Changing the key may result in the need to provide an upgrade for it. On the other hand, this portlet is also of `category.hidden`. Can we leave the decision for this key to Máté Thurzó as he's the author?